components/switch/stories/switch.stories.js

```diff
@@ -50,31 +52,78 @@ WithForcedColors.parameters = {
 };
 
 // ********* DOCS ONLY ********* //
-export const Emphasized = Template.bind({});
+/**
+ * Switches can either be selected or not selected. They cannot be in an indeterminate state (unlike
+ * [checkboxes](?path=/docs/components-checkbox--docs)). When a switch represents multiple values that are not
+ * identical (mixed values), the switch should appear as not selected.
+ */
+export const DocsDefault = DocsSwitchGroup.bind({});
+DocsDefault.storyName = "Default";
+DocsDefault.tags = ["!dev"];
+DocsDefault.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A switch in a disabled state shows that a selection exists, but is not available in that circumstance. This can be
+ * used to maintain layout continuity and communicate that an action may become available later.
+ */
+export const Disabled = DocsSwitchGroup.bind({});
+Disabled.tags = ["!dev"];
+Disabled.args = {
+	isDisabled: true
+};
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * Emphasized switches are optimal for forms, settings, and other scenarios where the switches need to be noticed. Not
+ * emphasized (gray) switches are optimal for application panels where all the visual components are monochrome in
+ * order to direct focus to the canvas.
+ */
+export const Emphasized = DocsSwitchGroup.bind({});
 Emphasized.tags = ["!dev"];
 Emphasized.args = {
 	isEmphasized: true,
-	label: "Switch label that is so long it wraps to the next line",
-	customStyles: {"max-width": "250px"}
 };
 Emphasized.parameters = {
 	chromatic: { disableSnapshot: true }
 };
 
-export const Checked = Template.bind({});
-Checked.tags = ["!dev"];
-Checked.args = {
-	isChecked: true
+/**
+ * When the label is too long for the horizontal space available, it wraps to form another line.
+ */
+export const WithLongerLabel = DocsSwitchGroup.bind({});
+WithLongerLabel.storyName = "Longer label";
+WithLongerLabel.tags = ["!dev"];
+WithLongerLabel.args = {
+	label: "Switch label that is so long it wraps to the next line",
+	customStyles: {"max-width": "250px"}
 };
-Checked.parameters = {
+WithLongerLabel.parameters = {
 	chromatic: { disableSnapshot: true }
 };
 
-export const Disabled = Template.bind({});
-Disabled.tags = ["!dev"];
-Disabled.args = {
-	isDisabled: true
+export const Sizing = (args, context) => Sizes({
+	Template,
+	...args,
+}, context);
```

Comment:
```suggestion
export const Sizing = (args, context) => Sizes({
	Template,
	withHeading: false,
	withBorder: false,
	...args,
}, context);
```

Using the no heading, no border variant reduces duplication for the docs page so only 1 sizing heading appears.